spec/models/payment_confirmation_spec.rb

```diff
@@ -43,8 +43,19 @@
       expect(second_payment.tax).to eq("162.8".to_d)
       expect(second_payment.net_pay).to eq("534".to_d)
       expect(second_payment.gross_pay).to eq("814.64".to_d)
+    end
+
+    it "populates the payroll run's confirmation_report_uploaded_by and sets the correct payment date" do
+      a_tuesday = Date.parse("2019-01-01")
+      the_following_friday = Date.parse("2019-01-04")
```

Comment:
these named variables make the spec really expressive 👍🏽. Maybe we should say "the following Friday" in the spec description. When I see "correct" in a spec description it feels like a smell as the spec description is an opportunity to tell me what "correct" in this case is without forcing me having to read the entire spec to figure that out.

Reply:
Yesh, that's a bad habit of mine. I'm trying to get out of it